resources/assets/lib/interfaces/user-json.ts

```diff
@@ -26,6 +26,10 @@ export default interface UserJson {
   is_online: boolean;
   is_supporter: boolean;
   last_visit: string | null;
+  page?: {
```

Comment:
this should probably be in `UserExtendedJson` but `CurrentUser` currently extends it and doesn't have `page` 🤔

Reply:
it's a `UserCompact` transformer include so it's part of `UserJson` (and it's not default include in `User` transformer either).

Reply:
it's only in the profile page which is what `UserExtendedJson` is (was) supposed to be for.